client/src/lib/result.js

```diff
@@ -1,32 +1,48 @@
 import { orderBy } from "./utils";
 import {
+  sum,
   projectedAverage,
   padSkipped,
   toMonotonic,
   isSkipped,
+  isComplete,
+  SKIPPED_VALUE,
+  NA_VALUE,
+  DNF_VALUE,
 } from "./attempt-result";
 
 /**
  * Returns a list of objects corresponding to result statistics - best and average.
  * The first statistic is the one that determines the ranking.
  * This is a common logic used in all result tables/dialogs.
  */
-export function orderedResultStats(eventId, format) {
+export function orderedResultStats(eventId, format, forecastView = null) {
   const { numberOfAttempts, sortBy } = format;
 
   if (!shouldComputeAverage(eventId, numberOfAttempts)) {
     return [{ name: "Best", field: "best", recordTagField: "singleRecordTag" }];
   }
 
-  const stats = [
+  var stats = (stats = [
```

Comment:
When defining variables, prefer to use `const` and `let` (when you reassign the name later). `var` is an old specifier and has its issues, which was the reason for introducing `const` and `let` :)

```suggestion
  let stats = [
```

(same for all other `var` usages)

Reply:
Ah didn't know that about var, let, and const. Thanks!